docs/GITFLOW.md

```diff
@@ -0,0 +1,39 @@
+## Git(hub) Flow:
+
+1. Checkout and pull latest `dev` branch
+2. Make your new feature branch following naming conventions
+3. Do your changes, and make pull request in github back to `dev` branch
+4. Ask for review, or(and) assign people on your pr to speed up process
+5. After receiving enough approves and testing, merge your pull request to `dev`
+
+`Note: make sure to pull dev frequent, to detect compatibility issues early`
+
+
+## Git naming conventions:
```

Comment:
To be sure that everyone follows branch naming rules and commit message rules - you can add [pre-push](https://git-scm.com/docs/githooks#_pre_push) and [commit-msg](https://git-scm.com/docs/githooks#_commit_msg) hooks.

And you can use these libs for example (or any other):
https://www.npmjs.com/package/@commitlint/cli
https://www.npmjs.com/package/validate-branch-name

What do you think about it? And of course, if you agree - it can be a separate task.

Reply:
yes, would be nice